kong/cmd/reload.lua

```diff
@@ -65,6 +66,7 @@ Options:
  -c,--conf        (optional string) configuration file
  -p,--prefix      (optional string) prefix Kong is running at
  --nginx-conf     (optional string) custom Nginx configuration template
+ --test-flags     (default 0)
```

Comment:
Maybe string will work better? For example "--test-flags coverage,no-fips,jit", or if we do not want it to be complex, just "--test-flags custom"

Reply:
yes! I had the same thoght this morning :) let's do that